tailwind.config.js

```diff
@@ -81,8 +89,15 @@ const placeholderColor = {
 
 // TODO: need review
 const buttonsTextColors = {
+  'controls-solid': 'var(--controls-text-solid, #FCFCFC)',
+  'controls-solid-disable': 'var(--controls-text-solid-disable, #333942)',
+  'controls-outlined': 'var(--controls-text-outlined, #FCFCFC)',
+  'controls-link-primary-hover':
+    'var(--controls-text-link-primary-hover, #3970DA)',
+  'controls-disable': 'var(--controls-text-disable, #5B6570)',
```

Comment:
we already have 'controls-disable' and it looks like second should be removed

```suggestion
  'controls-disable': 'var(--controls-text-disable, var(--controls-disable, #5B6570))',
```

Reply:
replace to  'controls-disable': 'var(--controls-disable, #5B6570)',